examples/Crate.elm

```diff
@@ -0,0 +1,305 @@
+module Crate exposing (main)
```

Comment:
This one doesn't come up as being renamed. Have you forgotten to delete the old `crate.elm`?

Reply:
Yes, Andrey the Crate one doesn't come up as being renamed, but I have no idea why. Sorry.

Reply:
In your branch https://github.com/RalfBarkow/webgl/tree/develop/examples you still have the old `crate.elm` and the new `Crate.elm`.